app/controllers/idv/inherited_proofing_cancellations_controller.rb

```diff
@@ -60,22 +65,10 @@ def session_go_back_path
       idv_session.go_back_path
     end
 
-    # AllowWhitelistedFlowStepConcern Concern overrides
+    # AllowlistedFlowStepConcern Concern overrides
 
     def flow_step_whitelist
```

Comment:
Can we rename to allowlist consistently?

```suggestion
    def flow_step_allowlist
```

Reply:
Good catch, done!